api/components/com_content/src/Controller/ArticlesController.php

```diff
@@ -68,6 +68,10 @@ public function displayList()
         if (\array_key_exists('state', $apiFilterInfo)) {
             $this->modelState->set('filter.published', $filter->clean($apiFilterInfo['state'], 'INT'));
         }
+        
+        if (\array_key_exists('featured', $apiFilterInfo)) {
```

Comment:
```suggestion

        if (\array_key_exists('featured', $apiFilterInfo)) {
```
Remove white space in empty line to fix code style check failing in drone.